package.json

```diff
@@ -7,7 +7,7 @@
     "autoprefixer": "^7.1.6",
     "css-mqpacker": "^6.0.1",
     "cssnano": "^3.10.0",
-    "moment": "2.3.x",
```

Comment:
Huh, but I thought 2.11 has the vulnerability?

Reply:
probably <2.11

Reply:
Ah woops I said `2.11` but was thinking `2.1` XD, got it